python/felis/cli.py

```diff
@@ -117,48 +117,56 @@ def create(
     Notes
     -----
     This command creates database objects from the Felis file. The
-    ``--create-if-not-exists`` or ``--drop-if-exists`` flags can be used to
-    create a new MySQL database or PostgreSQL schema if it does not exist
-    already.
+    ``--initialize`` or ``--drop`` flags can be used to create a new MySQL
```

Comment:
Not sqlite?

Reply:
SQLite databases are created automatically by SQLAlchemy if you provide a URL like `sqlite:///example.db` so the option isn't needed - right now it is just silently ignored if provided.

Reply:
I'll add some documentation which explains this.